iptables-wrapper-installer.sh

```diff
@@ -86,7 +108,11 @@ fi
 
 # Copy the wrapper.
 rm -f "${sbin}/iptables-wrapper"
-mv "${iptables_wrapper_path}" "${sbin}/iptables-wrapper"
+if [ -z "$no_cleanup" ]; then
+    mv "${iptables_wrapper_path}" "${sbin}/iptables-wrapper"
+else
+    cp "${iptables_wrapper_path}" "${sbin}/iptables-wrapper"
```

Comment:
You could also just always do "cp" rather than "mv" here, and then rm the original in the "# Cleanup" section below

Reply:
Done